rbd/group.go

```diff
@@ -110,7 +110,9 @@ func GroupImageAdd(groupIoctx *rados.IOContext, groupName string,
 		cephIoctx(groupIoctx),
 		cGroupName,
 		cephIoctx(imageIoctx),
-		cImageName)
+		cImageName,
+		C.uint32_t(0),
```

Comment:
This change is part of the [in-progress ceph PR](https://github.com/ceph/ceph/pull/53793/files) that has mirror group APIs

Reply:
Hmmm. I don't think that's going to be an acceptable kind of change to an existing public API. It would retroactively break working code. That PR is enormous but I guess I'll have to look through it and raise that concern over there. Normally, I'd expect a new api function to be added (say "foo_bar2" or whatever).

The fact that this is still touching an existing stable API is a warning sign.